judger/src/tester/exec.rs

```diff
@@ -637,6 +720,70 @@ impl TestSuite {
     }
 }
 
+/// Create a test case out of various configs.
+///
+/// This function is extracted from TestSuite::Run.
+/// TODO: Refactor this function.
+async fn create_test_case(
+    public_cfg: &JudgerPublicConfig,
+    test_root: &PathBuf,
```

Comment:
clippy: writing `&PathBuf` instead of `&Path` involves a new object where a slice will do.
`#[warn(clippy::ptr_arg)]` on by default
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#ptr_arg

Reply:
Done.